ts/src/bitget.ts

```diff
@@ -2210,11 +2210,12 @@ export default class bitget extends Exchange {
         await this.loadMarkets ();
         const currency = this.currency (code);
         const networkId = this.networkCodeToId (chain, code);
+        const precisionAmount = this.currencyToPrecision (code, amount, networkId);
```

Comment:
`currencyToPrecision` function accepts `networkCode` parameter, not the `networkId`. The `...Code` variable is internal to ccxt, unified one, whereas the `...Id` is usually an external variable. Here, networkId is the external network name